README.md

```diff
@@ -11,39 +11,60 @@
+secrets by reference. This not only allows us to automatically connect the services to each other, but it also allows
+Architect to build strict network policies to whitelist the traffic between these services. Now we won't have any work ahead
+of us to promote this stack from local dev all the way through to production!
 
-This example will show you the use-case for using Python on Architect leveraging the Django tutorial application – [Polls](//docs.djangoproject.com/en/4.0/intro/tutorial01/). In this example, we've written a component spec (the `architect.yml` file) that defines a component to run a Python based web application.
+[Learn more about the architect.yml file](//docs.architect.io/configuration)
 
-[Learn more about the architect.yml file](//docs.architect.io/components/architect-yml/)
+## Using the API
+This API implements basic CRUD functionality for a simple `Items` schema consisting of a `name` and a `rating` between 1 and 5.
+You could use it to gather data about anything you want to rate, from your favorite restaurants, movies, and more!
 
-## Running locally
+### The `Items` Schema:
+
+```
+  {
+    "name": "string",
+    "rating": "integer"
+  }
+```
+## Running Locally
+The `architect.yml` file is declarative, which allows the Architect Component it describes to be run in any environment,
+from local development all the way to production. Follow these steps to clone this repository and run the application
+locally.
 
-Architect component specs are declarative, so it can be run locally or remotely with a single deploy command:
+Once the deployment has completed, you can reach your new service by going to https://api.localhost.architect.sh.
 
 ```sh
 # Clone the repository and navigate to this directory
-$ git clone https://github.com/architect-templates/django.git
+$ git clone git@github.com:architect-templates/django.git
 $ cd ./django
 
-# Register the component to the local registry
-$ architect link .
-
-# Deploy using the dev command
-$ architect dev .
+# Deploy locally using the dev command
+$ architect dev architect.yml
 ```
 
-Once the deploy has completed, you can reach your new service by going to https://app.localhost.architect.sh/.
+## Deploying to the Cloud
+
+Want to try deploying this to a cloud environment? Architect's got you covered there, too! It only takes a minute to
+[sign up for a free account](https://cloud.architect.io/signup).
 
-Default values of `admin` and `password` have been set for the `/admin` page. To access or change the default admin user credentials, modify the parameters on the top of the `architect.yml` file.
+You can then [deploy the component](https://docs.architect.io/getting-started/introduction/#deploy-to-the-cloud) using the command below from the `flask` directory. Note that “example-environment” is the free environment that is created with your Architect account.
```

Comment:
Says `flask` directory  - i think this would be run from the root directory? Definitely not flask though 😛